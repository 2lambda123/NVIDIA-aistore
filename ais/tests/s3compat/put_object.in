echo "0123456789" > $OBJECT.txt
aws --endpoint-url http://localhost:8080/s3 s3 mb s3://$BUCKET
aws --endpoint-url http://localhost:8080/s3 s3 cp $OBJECT.txt s3://$BUCKET/$OBJECT // IGNORE
aws --endpoint-url http://localhost:8080/s3 s3 ls s3://$BUCKET | wc -l
aws --endpoint-url http://localhost:8080/s3 s3 cp s3://$BUCKET/$OBJECT $OBJECT_copy.txt  // IGNORE
cat $OBJECT_copy.txt
aws --endpoint-url http://localhost:8080/s3 s3 rm s3://$BUCKET/$OBJECT  // IGNORE
aws --endpoint-url http://localhost:8080/s3 s3 rb s3://$BUCKET
rm $OBJECT.txt // IGNORE
rm $OBJECT_copy.txt // IGNORE
