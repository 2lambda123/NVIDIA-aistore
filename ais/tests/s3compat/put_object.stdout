make_bucket: $BUCKET
1
0123456789
remove_bucket: $BUCKET
